app/controllers/application_controller.rb

```diff
@@ -8,16 +8,16 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_user!
```

Comment:
Its odd that this didn't do require_user before - was it done in each controller or something? If so have we made a few things now have auth that didn't before by accident here?

Reply:
Yes, it was done in each controller before. We are now following the best-practice way which is to enable it in ApplicationController then whitelist the actions you'd like to skip with:

```
skip_before_action :authenticate_user!, only: %i[timezone]
```

The behavior is consistent so we don't need to make a note in the migration docs.